lib/commands/debug.ts

```diff
@@ -18,7 +18,10 @@
 
 					this.debugService.debugStop().wait();
 
-					let applicationId = deviceAppData.device.isEmulator ? projectData.projectName : deviceAppData.appIdentifier;
+					let applicationId = deviceAppData.appIdentifier;
+					if (deviceAppData.device.isEmulator && deviceAppData.platform.toLowerCase() === this.$devicePlatformsConstants.iOS.toLowerCase()) {
```

Comment:
I've seen this code in many many places. Do you think that it's reasonable to extract it in a helper method somewhere ?